src/components/UserProfile/elements/TransitionReplace.jsx

```diff
@@ -36,20 +36,21 @@ class TransitionReplace extends React.Component {
   }
 
   onChildEntering(htmlNode) {
-    // subtract 1 from the height to improve shifts at the end of the animation
     this.setState({
-      height: htmlNode.offsetHeight - 1,
+      height: htmlNode.offsetHeight,
```

Comment:
This change and the identical one just removes a small hack that doesn't seem to be doing anything anymore.